options_handler.go

```diff
@@ -0,0 +1,35 @@
+package rocket
+
+import (
+	"reflect"
+
+	"github.com/dannypsnl/rocket/response"
+)
+
+type optionsHandler struct {
+	methods []string
+}
+
+func newOptionsHandler() *optionsHandler {
+	return &optionsHandler{
+		methods: make([]string, 0),
+	}
+}
+
+func (o *optionsHandler) addMethod(method string) *optionsHandler {
+	o.methods = append(o.methods, method)
+	return o
+}
+
+func (o *optionsHandler) build() *handler {
+	allowMethods := "OPTIONS"
+	for _, m := range o.methods {
+		allowMethods += ", " + m
+	}
+	return newHandler(reflect.ValueOf(func() *response.Response {
+		return response.New("").
+			Headers(response.Headers{
+				"Allow": allowMethods,
```

Comment:
you mean `Access-Control-Allow-Methods`?

Reply:
Nope, 
```
HTTP/1.1 200 OK
Allow: OPTIONS, GET, HEAD, POST
Cache-Control: max-age=604800
Date: Thu, 13 Oct 2016 11:45:00 GMT
Expires: Thu, 20 Oct 2016 11:45:00 GMT
Server: EOS (lax004/2813)
x-ec-custom-error: 1
Content-Length: 0
```